notebooks/iatraoui.ipynb

```diff
@@ -237,8 +293,26 @@
     "        _type_: A list of the RGB values contained in the data_array and their corresponding classes\n",
     "    \"\"\"\n",
     "\n",
-    "    ...  # TODO\n",
-    "    raise NotImplementedError\n"
+    "    data_array = data_array.stack(z=(\"x\", \"y\"))\n",
+    "    burnt_polygons = burnt_polygons.reshape(1830**2)\n",
```

Comment:
You can use `.reshape(-1)` as a way to flatten an array.

Benefits is that you don't need to know the target shape. This means less coupling of your code to the actual data. You could give an array of different shape eg `(5490, 5490)` and it would still works!

```python
In [1]: import numpy as np
   ...: 

In [2]: arr = np.arange(16).reshape(4,4)

In [3]: arr
Out[3]: 
array([[ 0,  1,  2,  3],
       [ 4,  5,  6,  7],
       [ 8,  9, 10, 11],
       [12, 13, 14, 15]])

In [4]: arr.reshape(-1)
Out[4]: array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15])

In [5]: arr2 = np.arange(2*3*5).reshape(2,3,5)

In [6]: arr2
Out[6]: 
array([[[ 0,  1,  2,  3,  4],
        [ 5,  6,  7,  8,  9],
        [10, 11, 12, 13, 14]],

       [[15, 16, 17, 18, 19],
        [20, 21, 22, 23, 24],
        [25, 26, 27, 28, 29]]])

In [7]: arr2.reshape(-1)
Out[7]: 
array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15, 16,
       17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29])
```